foreman_rh_cloud.gemspec

```diff
@@ -19,7 +19,6 @@ Gem::Specification.new do |s|
   s.test_files -= Dir['test/**/*.orig']
 
   s.add_dependency 'katello'
-  s.add_dependency 'redhat_access'
```

Comment:
Do we also need to update https://github.com/theforeman/foreman_rh_cloud/blob/master/config/Gemfile.lock.gh_test ?